### PR TITLE
ZK-4574: Clients.resize() doesn't work for hflex="min"

### PR DESCRIPTION
**Pull request by DevChu**

### CODE REVIEW

File: zk/src/archive/web/js/zk/widget.js [hunk 1]
```diff
@@ -1107,17 +1107,17 @@ new zul.wnd.Window({
 							//checking on (_binds[this.uuid] === this) as before does not work when
 							//nested inside native component. in this case the nested component
 							//is bound earlier, when the native component is reused (mount.js create())
+			if (this._cssflex)
```
Comment (plijyfes) [inline zk/src/archive/web/js/zk/widget.js:1110]: testing min->number and number->number should clear or not

**Review by plijyfes — CHANGES_REQUESTED** [review]

File: zktest/src/archive/test2/B95-ZK-4574.zul [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		</button>
+		<button label="2. Click it, you should see the blue div change to double height of red div (vflex to 2)">
+			<attribute name="onClick"><![CDATA[
+			div1.setVflex("2");
+		]]></attribute>
+		</button>
+		<button label="3. Click it, you should see the blue div change to same height of red div (vflex to 1)">
+			<attribute name="onClick"><![CDATA[
+			div1.setVflex("1");
+		]]></attribute>
```
Comment (scribetw) [inline zktest/src/archive/test2/B95-ZK-4574.zul:31]: Add 1 to ""

**Review by scribetw — CHANGES_REQUESTED** [review]

**Review by plijyfes — APPROVED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**